src/dynamicprompts/commands/variant_command.py

```diff
@@ -21,15 +22,18 @@ class VariantOption:
 class VariantCommand(Command):
     variants: list[VariantOption]
     min_bound: int = 1
-    max_bound: int = 1
+    max_bound: int = None
```

Comment:
This contradicts the type hint.
```suggestion
    max_bound: int | None = None
```